windows-driver-docs-pr/devtest/28719-banned-api-usage-use-updated-function-replacement.md

```diff
@@ -9,11 +9,146 @@ f1_keywords:
 # C28719
 
 
-warning C28719: Banned API Usage
+**warning C28719: Banned API Usage**\
+_Banned API Usage:  *function name* is a Banned API as listed in dontuse.h for security purposes._
 
-This warning indicates that a function is being used that has been banned, and has a more robust and secure replacement.
+This warning indicates that a function is being used that has been banned and has a more robust or secure replacement.
+A list of all banned functions covered by this error, why they are banned, and recommended replacements can be found after the following example: 
 
- 
+## Example 
+
+The following code generates this warning: 
+```
```

Comment:
See if you can get syntax highlighting working for this. I posted a suggestion in another PR.

Reply:
Added!